src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditContainer.tsx

```diff
@@ -38,6 +38,28 @@ interface WorkPackageEditContainerProps {
   exitEditMode: () => void;
 }
 
+export interface dataPayload {
```

Comment:
if we're exporting this to be used somewhere else, it needs a more descriptive name than just `dataPayload` because that could mean anything

try `WorkPackageEditFormPayload`